Makefile.Common

```diff
@@ -62,7 +62,7 @@ benchmark:
 
 .PHONY: addlicense
 addlicense:
-	@ADDLICENCESEOUT=`$(ADDLICENCESE) -y -c 'OpenTelemetry Authors' $(ALL_SRC) 2>&1`; \
```

Comment:
We don't want to use year in our files, so I suggest to do the same as in core https://github.com/open-telemetry/opentelemetry-collector/blob/main/Makefile.Common#L37 and add an empty string after `-y "" -c`

Reply:
Add the empty string, though it seems it with end up having two space between copyright and the holder.  Oops, you already have issue for it https://github.com/google/addlicense/issues/36